teuthology/task/install/deb.py

```diff
@@ -57,15 +64,20 @@ def _update_package_list_and_install(ctx, remote, debs, config):
     builder.install_repo()
 
     remote.run(args=['sudo', 'apt-get', 'update'], check_status=False)
-    remote.run(
-        args=[
+    install_cmd = [
```

Comment:
This seems like a good opportunity to start using this here: https://github.com/ceph/teuthology/blob/master/teuthology/packaging.py#L64

Reply:
@zmc Switching to that will change the way the install task currently behaves in one key respect. See https://github.com/ceph/teuthology/blob/master/teuthology/task/install/deb.py#L67

Due to this `'%s=%s' % (d, version) for d in debs`, the install task fails the test if apt cannot find the exact version that teuthology thinks is supposed to be installed.

Are you OK with that?

(I'm OK with it, because the equivalent RPM code does not behave in the same way.)

Reply:
Hmm, I'm actually not sure what to think now.

I'm leaning towards *not* making that change, if only to reduce the risk of regressions. Perhaps later on we implement and test what I suggested above...